web/profiles/custom/os2loop/themes/os2loop_theme/os2loop_theme.theme

```diff
@@ -11,3 +11,24 @@
 function os2loop_theme_preprocess(&$variables) {
   $variables['logopath'] = file_url_transform_relative(file_create_url(theme_get_setting('logo.url')));
 };
+
+/**
+ * Implements hook_preprocess_node().
+ *
+ * Add edit and deline links.
+ */
+function os2loop_theme_preprocess_field(&$variables) {
+  if ($variables["element"]["#field_name"] === "os2loop_post_comments" || $variables["element"]["#field_name"] === "os2loop_question_answers") {
+    $variables['current_user_name'] = $variables['user']->getDisplayName();
+  }
+}
```

Comment:
If we already have the current user available as `user` we should be able to `user.display_name` in the template and remove this hook implementation.